app/_src/kubernetes-ingress-controller/guides/services/grpc.md

```diff
@@ -129,9 +136,20 @@ grpcroute.gateway.networking.k8s.io/grpcbin created
 {% endnavtab %}
 {% navtab Ingress %}
 
-All routes are assumed to be either HTTP or HTTPS by default. We need to update the route to specify gRPC as the protocol by adding a `konghq.com/protocols` annotation.
+All routes and services are assumed to be either HTTP or HTTPS by default. We need to update the service to specify gRPC as the protocol by adding a `konghq.com/protocol` annotation.
+
+The annotation `grpcs` informs Kong that this service is a gRPC (with TLS) service and not a HTTP service.
+
+```bash
+kubectl annotate service grpcbin 'konghq.com/protocol=grpcs'
```

Comment:
Shouldn't this be `protocols`? https://docs.konghq.com/kubernetes-ingress-controller/latest/reference/annotations/#konghqcomprotocols

Reply:
https://docs.konghq.com/kubernetes-ingress-controller/latest/reference/annotations/#service-resource

<img width="954" alt="image" src="https://github.com/Kong/docs.konghq.com/assets/9593424/952148ec-05b5-4d84-a15e-5c3f4f3524ab">